src/types.d.ts

```diff
@@ -2,7 +2,9 @@ export interface Page {
   prefix: string;
   id: string;
 }
-
+export interface Authorization {
```

Comment:
No need to place it globally here. It should be somewhere in `AuthPage` folder. Probably separate file but you can also just import it.